app/src/main/contrib/components/Responsibilities/Overview/ResponsibilityOverviewDescription.tsx

```diff
@@ -95,15 +61,35 @@ export const ResponsibilityOverviewDescription = (props: ResponsibilityOverviewD
     }
 
     if (settings.isApplicantTouchstone(props.currentTouchstoneId)) {
```

Comment:
Old hardcoded stochastic touchstone no longer required. We could probably take out the JanuaryRfp touchstone logic too.